tuxemon/map_loader.py

```diff
@@ -208,7 +208,11 @@ def load(self, filename: str) -> TuxemonMap:
                 colliders = gids_with_colliders.get(gid)
                 if colliders is not None:
                     for obj in colliders:
-                        if obj.type and obj.type.lower().startswith(
+                        if obj.type is None:
+                            obj_type = obj.__getattribute__("class")
+                        else:
```

Comment:
Can you use `getattr(obj, "class")` ? It's more standard

Reply:
Done